desktop/libs/indexer/src/indexer/file_format.py

```diff
@@ -303,6 +304,14 @@ class ParquetFormat(FileFormat):
   _description = _("Parquet Table")
 
 
+class XLSXFormat(GrokkedFormat):
+  _name = "xlsx"
+  _description = _("XLSX File")
+  _args = [
+    CheckboxArgument("hasHeader", "Has Header")
+  ]
```

Comment:
Also we are not adding here `_extensions` here for XLSXFormat just like we have for JSON and CSV because internally it is getting converted to CSV?

Reply:
done